apps/docs/pages/guides/auth/social-login.mdx

```diff
@@ -41,6 +41,12 @@ Supabase supports a suite of social providers. Follow these guides to configure
       </a>
     </Link>
 ))}
+  
+## Provider Tokens
```

Comment:
@hf Should this be below the `</div>?

Reply:
Oh my bad. Didn't see the </div>

Reply:
ha, beat me to it

Reply:
LOL github does not allow me to write `</div>` :D :D :D

Reply:
@hf oh yes, happens to me all the time!